index.js

```diff
@@ -48,55 +48,37 @@ const loginBtnsEl = document.getElementById("login-btns")
 const logoutBtnEl = document.getElementById("logout-btn")
 const signUpBtn = document.getElementById("sign-up-btn")
 const signInBtn = document.getElementById("sign-in-btn")
-const xOutModalBtnsArr = document.querySelectorAll(".x-out")
 const buttonsArr = document.querySelector("#container").querySelectorAll(".btn")
 const createUserBtn = document.getElementById("create-user")
 const loginBtn = document.getElementById("login-btn")
 const logoutBtn = document.getElementById("log-out-btn")
 const signUpForm = document.getElementById("sign-up-form")
 const signInForm = document.getElementById("sign-in-form")
+const signUpWrapper = document.getElementById("sign-up-wrapper")
+const signInWrapper = document.getElementById("sign-in-wrapper")
 let isValid = false
 
-document.querySelectorAll(".login-btn").forEach(function(loginBtn) {
-    loginBtn.addEventListener("click", function() {
-        displayGenericModal(`${loginBtn.dataset.modalType}`)
-    })
-})
-
-signUpBtn.addEventListener("click", function() {
-    displayGenericModal("sign-up")
-})
-
-signInBtn.addEventListener("click", function() {
-    displayGenericModal("sign-in")
-})
-
-xOutModalBtnsArr.forEach(function(button) {
-    button.addEventListener("click", function(event) {
-        const currentModal = event.target.parentElement
-
-        hideModal(currentModal)
-
-        document.querySelectorAll(".login-btn").forEach(function(loginBtn) {
-            enableBtn(loginBtn)
-        })
-    })
+document.addEventListener("click", event => {
+    [...event.target.classList].includes('x-out') ? event.target.parentElement.parentElement.classList.toggle('hidden') :
```

Comment:
love this! Didn't know you could spread it out like that, or rather never considered.